libs/SalesforceReact/src/com/salesforce/androidsdk/reactnative/bridge/SmartSyncReactBridge.java

```diff
@@ -250,7 +250,10 @@ private void handleSyncUpdate(final SyncState sync, Callback successCallback, Ca
                     ReactBridgeHelper.invokeSuccess(successCallback, sync.asJSON());
                     break;
                 case FAILED:
-                    errorCallback.invoke("Sync failed");
+                    //errorCallback.invoke("Sync failed");
```

Comment:
Could you get rid of that commented code?

Reply:
Resolved in 5f766a5e25f7e5a2c4a1956e038e4dc5c31e8881